src/Blocks/components/paragraph/manifest.json

```diff
@@ -41,6 +45,14 @@
 			{
 				"label": "Small (18px)",
 				"value": "small"
+			},
+			{
+				"label": "Excerptional (16px)",
```

Comment:
Not sure we need descriptive names for font sizes 😂 

There is a list of font sizes on the net, and it's usually something like small, tiny, medium, default, large, etc. (plus comparatives larger, largest...). This gives us enough range 😄